backend/backend/celery_task.py

```diff
@@ -0,0 +1,25 @@
+import logging
+import os
+from typing import Any
+
+from celery import shared_task
+from utils.log_events import handle_user_logs
+
+from unstract.core.constants import LogEventArgument, LogProcessingTask
+
+logger = logging.getLogger(__name__)
+
+
+class TaskRegistry:
+    @staticmethod
+    @shared_task(
+        name=LogProcessingTask.TASK_NAME,
+    )
+    def log_consumer(**kwargs: Any) -> None:
```

Comment:
NIT: Please add a docstring highlighting the kwargs

Reply:
@chandrasekharan-zipstack Updated. I was missed entire docstring